### PR TITLE
Fix MS Edge when it scrolls to last added and selected HTMLElement

### PR DESCRIPTION
**Pull request by jaybekster**

### CODE REVIEW

**Comment by sudodoki** [comment]
LGTM

File: index.js [hunk 1]
```diff
@@ -20,6 +20,14 @@ function copy(text, options) {
     mark.textContent = text;
     // used to conserve newline, etc
     mark.style.whiteSpace = 'pre';
+
+    // position fixed because MS Edge scroll to upcoming seleceting node
+    // apart from this part it scrolls to the end of page
```
Comment (sudodoki) [inline index.js]: Not sure what was meant here. Maybe, `following code prevents scrolling to the end of the page`?

Comment (jaybekster) [inline index.js]: Yes, you re right

Comment (sudodoki) [inline index.js]: Can you please update the comment then?

**Comment by sudodoki** [comment]
Thanks for your contribution

**Comment by sudodoki** [comment]
@jaybekster can you test latest master on MS Edge? Got #28 after this changed and tried fixing it in 7e10898

**Comment by jaybekster** [comment]
Yeah, it still works well in Edge after this new one commit

**Comment by sudodoki** [comment]
Thanks for confirmation
3 июня 2016 г. 12:58 PM пользователь "Mikhail" <notifications@github.com>
написал:

**Pull request merged**